jerry-main/main-unix.c

```diff
@@ -121,6 +121,33 @@ print_help (char *name)
                       name);
 } /* print_help */
 
+static void
+print_unhandled_exception (jerry_value_t error_value)
+{
+  assert (jerry_value_has_error_flag (error_value));
+
+  jerry_value_clear_error_flag (&error_value);
+  jerry_value_t err_str_val = jerry_value_to_string (error_value);
+  jerry_size_t err_str_size = jerry_get_string_size (err_str_val);
+  jerry_char_t err_str_buf[256];
+
+  if (err_str_size >= 255)
+  {
+    const char msg[] = "[Long message]";
```

Comment:
This is just a simple shell, having such limitations are perfectly fine to me. Adding a truncated get could be an API extension, the challenge is breaking UTF-8 characters. But I don't think it will happen before 1.0.